website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -149,12 +149,12 @@ loadInitializers(App, config.modulePrefix);"
 
 ### Limit height
 
-Code content uses `auto` height by default but you can opt to set a `maxHeight` value to save space. Vertical scrolling is enabled as part of this feature allowing users to scroll vertically to view the overflowing content.
+Code content uses `auto` height by default but you can opt to set a `maxHeight` value to save space. If the content height exceeds the set max height, vertical scrolling is enabled to scroll vertically to view the overflowing content. A toggle button is rendered as well, which can be used to expand the height and show the Code Block content in its entirety.
```

Comment:
[suggestion] Removes the duplicate use of "vertical" in the same sentence and combines the last two sentences into one with some simplification. Not overly opinionated on that last part, but it read a little bit awkward to me.

```suggestion
Code content uses `auto` height by default but you can opt to set a `maxHeight` value to save space. If the content height exceeds the set max height, vertical scrolling is enabled view the overflowing content and a toggle button is displayed to expand the height and show the Code Block content in its entirety.
```